src/test/java/easton/EastonTest.java

```diff
@@ -0,0 +1,60 @@
+package easton;
+
+import easton.exception.DateTimeFormatException;
+import easton.exception.EmptyDescriptionException;
+import easton.exception.InvalidFormatException;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
```

Comment:
Based on [this guide](https://se-education.org/guides/tutorials/intellijCodeStyle.html#tweak-import-order), I think static imports should be ordered first.